src/app/common/transactions/stacks/generate-unsigned-txs.ts

```diff
@@ -78,7 +78,7 @@ function generateUnsignedContractDeployTx(args: GenerateUnsignedContractDeployTx
     nonce: initNonce(nonce)?.toString(),
     fee: new BN(fee, 10)?.toString(),
     publicKey,
-    postConditionMode,
+    postConditionMode: postConditionMode ?? PostConditionMode.Deny,
```

Comment:
Might be the cause of a bug https://github.com/leather-io/extension/issues/6117

Reply:
Thx @camerow I see the link to here now. I think he is passing `allow`, but maybe it isn't get set correctly thru the message passing. If you want to try and fix, feel free. I don't think this will impact what I'm still working on with the sip9 transfer.